<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogToAdd</class>
 <widget class="QDialog" name="DialogToAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>302</width>
     <height>324</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="surnameLabel">
            <property name="text">
             <string>Surname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="surnameEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="patronimycLabel">
            <property name="text">
             <string>Patronimyc</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="patronimycEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="yearLabel">
            <property name="text">
             <string>Year</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="yearEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="courseLabel">
            <property name="text">
             <string>Course</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="courseEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="groupNumberLabel">
            <property name="text">
             <string>Group number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="groupNumberEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="pathToPicturelabel">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Path to picture</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pickButton">
            <property name="text">
             <string>Pick picture</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="pathToPictureEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
